pkg/errors/kinds.go

```diff
@@ -4,3 +4,9 @@ package errors
 func IsNotFoundErr(err error) bool {
 	return Kind(err) == KindNotFound
 }
+
+// IsConfigErr returns true if the given err is a configuration error
+func IsConfigErr(err error) bool {
```

Comment:
we probably don't need this if we just do `errors.Is(err, errors.ConfigErrorKind)` or whatever kind name we'll eventually pick. 

But furthermore, I don't think we need a new config error kind, because we don't programmatically do anything about it anyway, we just log the error and exit.